1-sprint-mission/docker-compose.yml

```
@@ -0,0 +1,42 @@
+version: "3.8"
+
+services:
+  app:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: discodeit-app
+    environment:
+      - SPRING_PROFILES_ACTIVE=${SPRING_PROFILES_ACTIVE}
+      - SPRING_DATASOURCE_URL=${SPRING_DATASOURCE_URL}
+      - SPRING_DATASOURCE_USERNAME=${SPRING_DATASOURCE_USERNAME}
+      - SPRING_DATASOURCE_PASSWORD=${SPRING_DATASOURCE_PASSWORD}
+      - JVM_OPTS=${JVM_OPTS}
+      - SERVER_PORT=${SERVER_PORT}
+      - ACCESS_KEY=${AWS_S3_ACCESS_KEY}
+      - SECRET_KEY=${AWS_S3_SECRET_KEY}
+      - REGION=${AWS_S3_REGION}
+      - BUCKET=${AWS_S3_BUCKET}
+    ports:
+      - "80:8080"
```

포트 설정에 문제가 있는 것 같은데요,  
Dockerfile에서는 80포트를 노출하고 있어서 아래와 같이 수정하는게 맞을 것 같아요.  
```suggestion
      - "8080:80"
```